UM/Mesh/MeshBuilder.py

```diff
@@ -759,3 +773,30 @@ def addConvexPolygonExtrusion(self, xy_points, y0, y1, color=None):
         self.addQuad(v0, v1, v2, v3, color=color, normal = normal)
 
         return True
+    
+    def _onFileChanged(self, file_path):
+        if os.path.isfile(self._file_name) and self._file_name is file_path:
+            message = Message(i18n_catalog.i18nc("@info", "Would you like to reload {filename}?").format(filename = os.path.split(self._file_name)[0]),
```

Comment:
`i18n_catalog` doesn't exist. This code was never run?

Reply:
Look at the PR description:
`Note: This has been coded without testing inside Github's webpage.`

There are at the moment some things I need to do. Will care about this PR asap.

Reply:
In your defense, you're not the one who merged it without testing... But it is better practice to only make PRs with tested code.

Reply:
Well it's been merged now so I need to take care of it...

Reply:
@Ghostkeeper: Alright, thanks!
@fieldOfView: My intention was to get things moving. So I began to make some code and share it as a PR. But you are right, next time I should practically test it.
Hope it will be much easier now since the PPA is working again.

Reply:
aww... :-/ (reverting is also an option)

Reply:
Yep, and then I have the chance for a practical PR :grimacing:

Reply:
Blame where blame is due. It was me merging it without testing. To be honest though I'm always still a little bit fuzzy about this process with respect to Jira. After all... this issue was caught by merging it and then running tests on it, right?

I will discuss with the team tomorrow to clarify my understanding so we don't run into this again.

Reply:
You can merge the branch into a local branch and test it locally before merging into master.

Reply:
@ianpaschal .. and it was me who share this ugly untested code.
Yep, what it definitely needs are not only builds and tests on master branches but also tests on PRs and all other branches. I suggested to use Travis for this in the past, but Ultimaker went for their own Jenkins solution.
Don't blame yourself on this. As I said I also have my part on this. Normally I'm not committing untested code (and make fun of copyrights), therefore let's be fair and say be both f**ked up. 😉